packages/features/calendars/DestinationCalendarSelector.tsx

```diff
@@ -102,7 +102,9 @@ const DestinationCalendarSelector = ({
     query.data.connectedCalendars.map((selectedCalendar) => ({
       key: selectedCalendar.credentialId,
       label: `${selectedCalendar.integration.title?.replace(/calendar/i, "")} (${
-        selectedCalendar.primary?.name
+        selectedCalendar.primary?.integration === "office365_calendar"
```

Comment:
<img width="665" alt="Screenshot 2022-12-13 at 8 03 08 PM" src="https://user-images.githubusercontent.com/53316345/207361238-d067b56f-162f-48fa-a8ac-4601ead39b41.png">

this will also display email in the options